builtin/mainmenu/tab_local.lua

```diff
@@ -308,6 +313,9 @@ if enable_gamebar then
 				menudata.worldlist:set_filtercriteria(game.id)
 				core.set_topleft_text(game.name)
 				mm_texture.update("singleplayer",game)
+				core.sound_stop_all()
+				local _,_,technical_game_name = string.find(game.gamemods_path, ".*\\(.*)\\.*")
+				core.sound_play(technical_game_name, true)
```

Comment:
How about `game.id`, or does it not exist in this context?

Reply:
Good catch, yeah that local "game" has an id, making the change now, new commit (or do we at minetest prefer single commit PR's?)

Reply:
Does not matter. The PR will be squashed on merge anyway.